_includes/disqus_comments.html

```diff
@@ -0,0 +1,19 @@
+{% if page.comments %}
+  <div id="disqus_thread"></div>
+  <script>
+    var disqus_config = function () {
+      this.page.url = '{{ site.url }}{{ page.url }}';
+      this.page.identifier = '{{ site.url }}{{ page.url }}';
```

Comment:
As part of my newest commit (https://github.com/jekyll/minima/pull/21/commits/7f5ec66c334555b0155549c1d07adaf1a09f9030) I changed this behavior. Please let me know if that's acceptable. 

I don't have much experience with Jekyll + GH Pages, but I'm assuming if `site.url` is not present then I can use `site.github.url` as a fallback (?)